src/table.ts

```diff
@@ -426,19 +426,6 @@ class Table {
   drop(gaxOptions?: CallOptions): Promise<DropTableResponse>;
   drop(callback: DropTableCallback): void;
   drop(gaxOptions: CallOptions, callback: DropTableCallback): void;
-  /**
-   * @typedef {array} DropTableResponse
-   * @property {Operation} 0 An {@link Operation} object that can be used to check
-   *     the status of the request.
-   * @property {object} 1 The full API response.
-   */
-  /**
-   * @callback DropTableCallback
-   * @param {?Error} err Request error, if any.
-   * @param {Operation} operation An {@link Operation} object that can be used to
-   *     check the status of the request.
-   * @param {object} apiResponse The full API response.
-   */
```

Comment:
I'm not sure why this is removed.

Reply:
It is been already define at:
https://github.com/googleapis/nodejs-spanner/blob/ca07a9067bd2259a716ed54aa0102e1fff19e891/src/table.ts#L270

Reply:
Thanks for the clarification.